### DISCUSSION TITLE
Why text wrap in docx

### DISCUSSION THREAD

**Discussion by ericosmic** [pymupdf/PyMuPDF#2511] [Looking for help]
I have some pdf files which some line text will be auto wrap after transfer into docx file,  i
orginal pdf :
![image](https://github.com/pymupdf/PyMuPDF/assets/35327931/9b3a686b-3ec6-4aa7-89e7-1b8f9bc128a0)
resulted docx:
![image](https://github.com/pymupdf/PyMuPDF/assets/35327931/cb4ae0a5-782c-4952-a2b0-4936e368cf3a)

anyone know what reson recall this.  
The problem pdf files :
[zf100.pdf](https://github.com/pymupdf/PyMuPDF/files/11934756/zf100.pdf)

**Comment by JorjMcKie** [comment]
This must be the fault of the software you use for converting PDF -> DOCX.
It looks like that not the original fonts are being used, so character widths may be different for the font used in the Word document.
In an effort to not exceed the horizontal space covered by "000100" then causes the last character to land on a new line.

**Reply by ericosmic** [reply]
I don't know what parameter set make it can't input word in the right of '00010', why would it force to wrap?

**Reply by JorjMcKie** [reply]
I do not know which conversion tool you used, so I cannot tell either.
